service/build.gradle

```diff
@@ -40,22 +43,25 @@ dependencies {
 	implementation 'org.apache.commons:commons-lang3:3.12.0'
 
 	// Test
-	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation("org.springframework.boot:spring-boot-starter-test") {
+		exclude group: "com.jayway.jsonpath", module: "json-path"
+	}
+	testImplementation("com.jayway.jsonpath:json-path:2.5.0") {
+		exclude group: "net.minidev", module: "json-smart"
+	}
+	testImplementation 'net.minidev:json-smart:2.4.1'
 	testImplementation 'org.junit.jupiter:junit-jupiter-params:5.7.0'
-	testImplementation 'com.github.tomakehurst:wiremock:2.27.2'
+	testImplementation "com.github.tomakehurst:wiremock:2.27.2"
 	testImplementation 'uk.org.webcompere:system-stubs-jupiter:1.2.0'
 	testImplementation "org.assertj:assertj-core:3.18.1"
 
 	constraints {
+		implementation('commons-io:commons-io:2.7') {
+			because 'ca.uhn.hapi.fhir depends on it'
```

Comment:
I see there's a precedent for this kind of explanation, but I think it would be better to explain why we insist on this version - "earlier versions had such and such a vulnerability", or "newer versions are incompatible because xyz" etc.

Reply:
turns out these are no longer needed, I have now removed